README.md

```diff
@@ -3,6 +3,7 @@
 ## Aqueduct: Prediction Infrastructure for Data Scientists
 
 [![Downloads](https://pepy.tech/badge/aqueduct-ml/month)](https://pypi.org/project/aqueduct-ml/)
+[![Slack](https://img.shields.io/static/v1.svg?label=chat&message=on%20slack&color=27b1ff&style=flat)](https://aqueductusers.slack.com)
```

Comment:
I think we want this to be our invite link instead: https://join.slack.com/t/aqueductusers/shared_invite/zt-11hby91cx-cpmgfK0qfXqEYXv25hqD6A